internal/common/vote/vote.go

```diff
@@ -0,0 +1,192 @@
+package vote
+
+import (
+	"math/rand"
+	"time"
+)
+
+func GetVotesForRule(ruleID string, NumOFIslands int) map[int][]int {
```

Comment:
`NumOfIslands` for PascalCase, not `NumOFIslands `

Reply:
Sorry...

Reply:
Function arguments should be camelCase, so it should be `numOfIslands`